packages/calypso-e2e/src/lib/blocks/block-flows/subscribe.ts

```diff
@@ -21,10 +16,14 @@ export class SubscribeFlow implements BlockFlow {
 	 */
 	async validateAfterPublish( context: PublishedPostContext ): Promise< void > {
 		// Is there an interactive email field?
-		const emailInputLocator = context.page.locator( selectors.emailInput );
+		const emailInputLocator = context.page
+			.getByRole( 'main' )
```

Comment:
This is similar to "anchoring" that I would like us to use in more POMs, where the topmost common parent is chosen as the "anchor" and is used to narrow the selection for all other queries. 

In this case, we are interested in the published content - in other words, only the part of the page affected by the editor, ignoring the header/footer/comment and etc.

Reply:
Makes sense! Great idea.